.rtd-environment.yml

```diff
@@ -11,5 +11,6 @@ dependencies:
     - matplotlib
     - numpy
     - requests
+    - scipy
```

Comment:
good point, we need to add it to the `[docs]` section in setup.cfg, too.

Reply:
done